src/libstd/sys/windows/ext/ffi.rs

```diff
@@ -42,11 +56,29 @@ impl OsStringExt for OsString {
 /// Windows-specific extensions to `OsStr`.
 #[stable(feature = "rust1", since = "1.0.0")]
 pub trait OsStrExt {
-    /// Re-encodes an `OsStr` as a wide character sequence,
-    /// i.e. potentially ill-formed UTF-16.
+    /// Re-encodes an `OsStr` as a wide character sequence, i.e. potentially
+    /// ill-formed UTF-16.
+    ///
+    /// This is lossless: calling [`OsString::from_wide()`] and then
```

Comment:
very minor, but can you remove all the `()` parens attached to method/function names in these docs? for the time being, the current policy is to remove them in documentation. see also https://github.com/rust-lang/rust/pull/40456